src/lib/y2packager/clients/inst_productsources.rb

```diff
@@ -140,9 +140,9 @@ def main
         return :auto
       end
 
-      # set release version for update as for newly added repositories
+      # Set the proper release version for newly added repositories
       # we want to use new product and not old
```

Comment:
Maybe this comment could be updated/improved, but right now I have no inspiration :) Do you have any idea? Thanks!